recipe/conda_build_config.yaml

```diff
@@ -1,5 +1,5 @@
 MACOSX_SDK_VERSION:         # [osx]
-  - 12.3                    # [osx]
+  - 11.3                    # [osx]
```

Comment:
Why? I think this was useful to ensure that certain hardware optimizations were available.